src/rdkafka_cgrp.c

```diff
@@ -2742,26 +2747,30 @@ static void rd_kafka_cgrp_group_leader_reset (rd_kafka_cgrp_t *rkcg,
 
 
 /**
- * @brief Group is rebalancing, trigger rebalance callback to application,
+ * @brief Initiate a rebalance. Trigger rebalance callback to application,
  *        and transition to INIT state for (eventual) rejoin.
  */
 static void rd_kafka_cgrp_rebalance (rd_kafka_cgrp_t *rkcg,
+                                     rd_bool_t assignment_lost,
                                      const char *reason) {
 
         rd_kafka_dbg(rkcg->rkcg_rk, CONSUMER|RD_KAFKA_DBG_CGRP, "REBALANCE",
-                     "Group \"%.*s\" is rebalancing in "
-                     "state %s (join-state %s) %s assignment: %s",
+                     "Group \"%.*s\" rebalance initiated in "
```

Comment:
dito

Reply:
yes, good point. what i was trying to capture is that we have the client initiating a revoke. I think the wording "group ... is rebalancing in state" is not precise / informative enough - if i read that in the logs, i'm going to wonder exactly what it means. if i read that the client is initiating a revoke, i have a very good idea what that means. i will change the wording and see what you think.

Reply:
... the client may be acting under instruction from the group coordinator, but it also may not (unsubscribe, or assignment lost).

Reply:
i've reverted this because i'm struggling to work out what to do and the COOPERATIVE protocol is going to put this in a different perspective again, so will defer doing anything until making those changes.